### ISSUE TITLE
IPFS Icon Request

### ISSUE THREAD

**Issue by cagataykandaz** [walkxcode/dashboard-icons#314]
### Type

Icon request

### Info

hi could you please add ipfs icon too, thanks!

**Comment by walkxcode** [comment]
All issues are temporarily on hold. Please refer to #316 for more info.

**Comment by walkxcode** [comment]
Hey, this icon has been added in the `new-standards` branch and will be merged once it is done!

https://github.com/walkxcode/dashboard-icons/blob/new-standards/png/ipfs.png
https://github.com/walkxcode/dashboard-icons/blob/new-standards/png/ipfs.svg
